gui-wm/wayfire/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <longdescription>
+        compiz like eye-candy 3D wayland, with low memory footprint.
+        Aims to bring desktop features such as 3D cube, fire burn,
+        fisheye, wrot and all other desktop effects to wayland.
+    </longdescription>
+    <maintainer type="person">
+        <email>gentoo@aisha.cc</email>
+        <name>Aisha Tammy</name>
+    </maintainer>
+    <maintainer type="project">
+        <email>proxy-maint@gentoo.org</email>
+        <name>Proxy Maintainers</name>
+    </maintainer>
+    <use>
+        <flag name="gles2">use libglvnd to do animations more efficiently</flag>
+        <flag name='wf-config'>use the system-wide wf-config, else build it as a submodule. DO NOT DISABLE UNLESS YOU KNOW WHAT YOU ARE DOING.</flag>
+        <flag name='wlroots'>use the system wide wlroots, else build it as a submodule. DO NOT DISABLE UNLESS YOU KNOW WHAT YOU ARE DOING.</flag>
+        <flag name='debug'>add debugging symbols for error reporting. ENABLE ALL USE FLAGS BEFORE REPORTING BUGS.</flag>
```

Comment:
I think I said this before but do you need USE flags which shouldn't be enabled? Also, people usually call these system-XYZ.

Reply:
unfortuntately yes, we do need them, for doing error reporting.

I forgot to flip the description to mention that you should disable these before reporting errors.

I tried to explain here: https://github.com/gentoo/gentoo/pull/15050#discussion_r425983749

Reply:
also for the use flag change, will definitely change that up. thanks !

Reply:
fixed